wire-ios-data-model/Tests/Source/Model/Conversation/ZMConversationTests+Teams.swift

```diff
@@ -93,33 +93,16 @@ final class ConversationTests_Teams: ZMConversationTestsBase {
         // given
         let oneOnOne = ZMConversation.insertNewObject(in: uiMOC)
         oneOnOne.conversationType = .oneOnOne
-        oneOnOne.connection = .insertNewObject(in: uiMOC)
-        oneOnOne.connection?.status = .accepted
+
         let userOutsideTeam = ZMUser.insertNewObject(in: uiMOC)
-        oneOnOne.connection?.to = userOutsideTeam
+        userOutsideTeam.connection = ZMConnection.insertNewObject(in: uiMOC)
+        userOutsideTeam.connection?.status = .accepted
+        userOutsideTeam.oneOnOneConversation = oneOnOne
 
         // then
         XCTAssertEqual(userOutsideTeam.oneToOneConversation, oneOnOne)
     }
 
-    func testThatItReturnsTeamConversationForOneOnOneConversationWithTeamMember() {
```

Comment:
I don't think this test makes much sense, there should be a connection with a team member.